packages/jss/src/utils/escape.js

```diff
@@ -1,15 +1,10 @@
-const env = process.env.NODE_ENV
-
-const escapeRegex = /([[\].#*$><+~=|^:(),"'`])/g
-
 export default str => {
   // We don't need to escape it in production, because we are not using user's
   // input for selectors, we are generating a valid selector.
-  if (env === 'production') return str
+  if (process.env.NODE_ENV === 'production') return str
 
-  if (CSS && CSS.escape) {
-    return CSS.escape(str)
-  }
+  const escapeRegex = /([[\].#*$><+~=|^:(),"'`])/g
```

Comment:
Should the regexp be moved outside of the function to provide better pers in dev?

Reply:
Also, we uses `/([[\].#*$><+~=|^:(),"'`\s])/g;` on Material-UI side to solve some other edge cases.

Reply:
I usually do that, but actually engine already does optimizations there, its not really necessarily and it won't be removed from the bundle afaik, if it does get removed we can move it to the top

Reply:
will update the regex with yours, it fixes the issue with spaces we already got reported and backticks

Reply:
oh just realized you don't have the backtick in the regex, you should add it too

Reply:
just looked again yes, I already added it too!

Reply:
our regexes are unified now, does mui has tests for all cases that regex covers? If yes I would copy them to make sure we don't break them by accident